server/src/main/java/com/linecorp/centraldogma/server/metadata/MetadataService.java

```diff
@@ -100,10 +99,8 @@ public class MetadataService {
     public MetadataService(ProjectManager projectManager, CommandExecutor executor) {
         this.projectManager = requireNonNull(projectManager, "projectManager");
         this.executor = requireNonNull(executor, "executor");
-        metadataRepo = new RepositorySupport<>(projectManager, executor,
-                                               entry -> convertWithJackson(entry, ProjectMetadata.class));
-        tokenRepo = new RepositorySupport<>(projectManager, executor,
-                                            entry -> convertWithJackson(entry, Tokens.class));
+        metadataRepo = new RepositorySupport<>(projectManager, executor, ProjectMetadata.class);
```

Comment:
In fact, `ProjectMetadata` is already cached in `DefaultProject` and automatically refreshed when the raw data is updated.

Reply:
What do you think of replacing the implementation of `fetchMetadata0()` with `projectManager.get(projectName).metadata()`?

Reply:
I can't replace it completely because fetching is still used here: https://github.com/line/centraldogma/blob/b9cebef958d6100f7da7f16da53c72d474ab888c/server/src/main/java/com/linecorp/centraldogma/server/metadata/MetadataService.java#L170
Let me add `getMetdata` instead. 😉